js/main.js

```diff
@@ -3,25 +3,99 @@
 
 var form = document.querySelector('form');
 var photoUrl = document.querySelector('#url');
-var img = document.querySelector('.placeholder');
+var img = document.querySelector('.form-img');
+var ul = document.querySelector('.entries');
 
 function photoUrlHandler(event) {
   img.setAttribute('src', photoUrl.value);
 }
+var entriesObj;
 
 function saveButtonHandler(event) {
   event.preventDefault();
   var title = form.elements.title.value;
   var url = form.elements.url.value;
   var notes = form.elements.notes.value;
-  var entriesObj = { title, url, notes };
+  entriesObj = { title, url, notes };
   entriesObj.entryId = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(entriesObj);
   img.setAttribute('src', 'images/placeholder-image-square.jpg');
   form.reset();
+  entriesContainer.className = 'container';
+  formContainer.className = 'container hidden';
+  addEntry(entriesObj);
+  localStorage.setItem('current', 'entries-current');
 }
 
 photoUrl.addEventListener('input', photoUrlHandler);
 
 form.addEventListener('submit', saveButtonHandler);
+
+function addEntry(entry) {
+  var entryLi = document.createElement('li');
+  var row = document.createElement('div');
+  var columnHalf = document.createElement('div');
+  var img = document.createElement('img');
+  var columnHalf2 = document.createElement('div');
+  var titleH2 = document.createElement('h2');
+  var notesP = document.createElement('p');
+  ul.appendChild(entryLi);
+  entryLi.appendChild(row);
+  row.appendChild(columnHalf);
+  columnHalf.appendChild(img);
+  row.appendChild(columnHalf2);
+  columnHalf2.appendChild(titleH2);
+  columnHalf2.appendChild(notesP);
+  ul.className = 'entries';
+  entryLi.className = 'entry';
+  row.className = 'row';
+  columnHalf.className = 'column-half';
+  columnHalf2.className = 'column-half';
+  titleH2.className = 'title-h2';
+  notesP.className = 'notes-p';
+  img.setAttribute('src', entry.url);
+  titleH2.textContent = entry.title;
+  notesP.textContent = entry.notes;
+  if (entry === entriesObj) {
+    ul.prepend(entryLi);
+  }
+}
```

Comment:
You want this function to `return` the DOM tree for this entry, so you can `append` it where the function is being called.